README.rst

```diff
@@ -1,20 +1,26 @@
-****************
+
+******************************
 django CMS Version Locking
-****************
+******************************
 
-## Explanation
-The version-locking addon is intended to modify the way djangocms-versioning works.  
-* The primary change is that it locks a version to its author when a draft is created. 
-* That version becomes automatically unlocked again once it is published. 
-* The lock prevents editing of the file by anyone other than the author.
-* Locks can be removed by a user with the correct permission
-* Unlocking an item sends a notification to the author to which it was locked.
-* Manually unlocking a version does not lock it to the unlocking user, nor does it change the author.
-* The Version admin view for each content-type registered with Version-locking will be edited to add in lock icons / buttons UI to the Actions column.
+Explanation
+------------
+The version-locking addon is intended to modify the way djangocms-versioning works in the following way: 
 
+ - The primary change is that it locks a version to its author when a draft is created. 
+ - That version becomes automatically unlocked again once it is published. 
+ - The lock prevents editing of the file by anyone other than the author.
+ - Locks can be removed by a user with the correct permission
+ - Unlocking an item sends a notification to the author to which it was locked.
+ - Manually unlocking a version does not lock it to the unlocking user, nor does it change the author.
+ - The Version admin view for each content-type registered with Version-locking will be edited to add in lock icons / buttons UI to the Actions column.
```

Comment:
I don't think "version admin view" is particularly clear. There are many version admin views really. I think the view you're referring to here we often call "the version table", but that won't be clear to anybody who is new to this either. If it's important to specify this (I'm not sure it is in the README?), I think two screenshots - of how a project looks like with locking enabled and how it looks like without would explain this better.

Reply:
Can one add screenshots to a markdown / rst document?